USERS.md

```diff
@@ -83,6 +83,12 @@ Users (Alphabetically)
       L: https://cloud.arangodb.com
       Q: @ewoutp @Robert-Stam
 
+    * N: Ascend.io
+      D: Ascend.io is using Cilium as a consistent CNI for our Data Automation Platform on GKE, EKS, and AKS.
+      U: Transparent Encryption, Overlay Networking, Cluster Mesh, Egress Gateway, Network Policy, Hubble
```

Comment:
I'm guessing Transparent Encryption is with WireGuard, right?

Reply:
We actually initially implemented with IPSec, but it looks like we're likely to transition to wireguard soon since the clouds all bumped their kernels to 5.10 with k8s 1.24 (partially because of incompatibility between ip-sec and some other features, I forget if it was cluster mesh or egress gateway that was the forcing function)

Reply:
Thanks for the answer!

> partially because of incompatibility between ip-sec and some other features, I forget if it was cluster mesh or egress gateway that was the forcing function

Probably egress gateway because IPsec is incompatible with KPR and egress gateway requires KPR. We're working on removing that limitation for IPsec.